crates/ide_completion/src/item.rs

```diff
@@ -122,7 +122,7 @@ impl fmt::Debug for CompletionItem {
     }
 }
 
-#[derive(Debug, Clone, Copy, Ord, PartialOrd, Eq, PartialEq, Default)]
```

Comment:
These weren't used, because `CompletionRelevance` should only be compared by score. I could re-implement them in terms of score, but that would require updating the `Eq` impl which we use in a different way, so it would broaden the scope of this PR a bit. 

The reason I removed these at all in this PR is because it didn't make sense to me to derive them on the new enum `CompletionRelevanceTypeMatch`.